js/src/wrappers/vercel/exporter.ts

```diff
@@ -0,0 +1,697 @@
+import type { CoreAssistantMessage, CoreMessage, ToolCallPart } from "ai";
+import type { AISDKSpan } from "./exporter.types.js";
+import { Client, RunTree } from "../../index.js";
+import { KVMap, RunCreate } from "../../schemas.js";
+import { v5 as uuid5 } from "uuid";
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type AnyString = string & {};
+
+type LangChainMessageFields = {
+  content:
+    | string
+    | Array<
```

Comment:
Are we typing this too strongly?

Reply:
This actually relaxes the string type to permit auto-complete while allowing other strings